test/react_client/event_helpers_test.dart

```diff
@@ -157,11 +157,11 @@ main() {
     });
 
     test('wrapNativeMouseEvent', () {
-      var nativeMouseEvent = MockMouseEvent();
-      var currentTarget = DivElement();
-      var target = DivElement();
-      var relatedTarget = DivElement();
-      var calls = <String>[];
+      final nativeMouseEvent = MockMouseEvent();
```

Comment:
This test also fails for `nativeMouseEvent.defaultPrevented` but with this error instead:
```
 Expected a value of type 'bool', but got one of type 'Null'
  dart:sdk_internal                as
  event_helpers_test.dart 2066:64  get defaultPrevented
  event_helpers_test.dart 169:28   <fn>
```
Which is a bit confusing since event being operated on is a `MockMouseEvent` and not a `MockKeyboardEvent` as per the stack trace.